Automattic/wp-calypso
# Unpin packages (Part 5)

#### Changes proposed in this Pull Request

* Unpin all dependencies in `client/package.json` (

Changes in the dependency tree:

```diff
< Before
> After
---
< wp-calypso-monorepo@0.17.0 wp-calypso@0.17.0 events@3.0.0
---
> wp-calypso-monorepo@0.17.0 wp-calypso@0.17.0 events@3.1.0
3063614c3063614
< wp-calypso-monorepo@0.17.0 wp-calypso@0.17.0 fast-json-stable-stringify@2.0.0
---
> wp-calypso-monorepo@0.17.0 wp-calypso@0.17.0 fast-json-stable-stringify@2.1.0
```

Changelogs from the affected packages:
* [events](https://github.com/Gozala/events/blob/master/History.md)
* [fast-json-stable-stringify](https://github.com/epoberezkin/fast-json-stable-stringify/releases)

#### Testing instructions

* Verify all tests pass
* Verify there are no new packages introduced (other than the two specified above): run `find . -type d -name node_modules | xargs rm -fr && yarn && NODE_OPTIONS=--max_old_space_size=8192 yarn run effective-module-tree -o list` in this branch and master (tested with commit f633e90c8d), save the output to a file and diff them.
